### DISCUSSION TITLE
Welcome to Pagirinya-Satellite-House Discussions!

### DISCUSSION THREAD

**Discussion by gibertgibs** [The-Youth-Empowerment-Foundation-Adju/Pagirinya-Satellite-House#2] [Q&A]
As the project continues, Imagine your community is running such a project but the youth in your community are too reluctant to change. They have without a clear reason chosen not to participate in any of the activities of the project or even accepted to participate but what they are doing doesn't favor what they are being taught by their trainers during their training sessions.
As the director, Program Manager, or any other resilient personality in the community, what will you do such kind of people or what measures will you use to bring them back to a discipline manner?

**Comment by timmwille** [comment]
>Well discipline only works in a short term and requires constant watch-over. What rather works is one-by-one sit-down, listening to them what future they have in mind and then draw the picture of how their participation can help shape this future. → plus, they need a specific task bound to their personal interest and trusted in responsibility and you will be there to guide when necessary. Complains, penalty and negative words wont bring you anywhere if volunteers are at work 
>→ **positive vibes, clear structure and statements and personal attachment to the project and in some cases achievement and personal growth will bring astonishing outcomes. Make their light shine bright!**


This in my experience is not easy to frame and construct, needs time, patience and a good mood yourself. 
This in my opinion also makes the difference in being a leader and coach instead of a ruler.  

PS: find those proactive ones that just need a little uplift first, don't waste too much time on those, who have all the excuses, they will join over time, when they see the fun and light shine. The beginning is always the hardest part, find allies. :v: 

![grafik](https://user-images.githubusercontent.com/8081055/107506403-529da500-6b9e-11eb-8850-f258128e367c.png)

**Reply by gibertgibs** [reply]
Perfect... WILL GO THROUGH D:-

**Reply by vugawilliam** [reply]
It's always take time to change someone, however learning is process and I believe collerbation and action will definitely change some life after he or she has seen the impact of any activities that connect and bring people together.

**Reply by gibertgibs** [reply]
Perfect... will analyse all.

**Reply by vugawilliam** [reply]
You know changing someone mindset take alot of energy and believe in  what you are doing to the community and always work hard to change chellanges as an opportunity to achieve your goals.

**Reply by chardso** [reply]
My teacher used to tell me never give up ,don't ignore any of them,table discussions, they will understand with time,same like teaching in the class,you will have those who will learn first and slow learners.
